src/main/java/org/broadinstitute/hellbender/tools/spark/pathseq/PSFilter.java

```diff
@@ -213,6 +223,14 @@ private static GATKRead clearReadAlignment(final GATKRead read, final SAMFileHea
 
         if (!filterArgs.skipFilters) {
 
+            //Adapter trimming
+            reads = reads.map(new ReadTransformerSparkifier(new IlluminaAdapterTrimTransformer(filterArgs.maxAdapterMismatches, filterArgs.minAdapterLength, ADAPTER_SEQUENCES)));
+
+            //Apply simple repeat masking
+            //See "Low-complexity DNA and simple repeats" at http://www.repeatmasker.org/webrepeatmaskerhelp.html
+            reads = reads.map(new ReadTransformerSparkifier(new SimpleRepeatMaskTransformer(29, 29, 30)));
```

Comment:
If these numbers may need to be tweaked, it would be good to add them to filterArgs. If they're final numbers, it may still be good to name them as constants (e.g. shortWindowLen, maxShortWindowGCCounts or similar, so that it's easier to figure out what they do).